folio-spring-system-user/src/main/java/org/folio/spring/service/PrepareSystemUserService.java

```diff
@@ -67,7 +67,7 @@ private void sanityCheck() {
       systemUserService.authSystemUser(folioExecutionContext.getTenantId(),
                                         systemUserProperties.username(), systemUserProperties.password());
       log.info("System user authenticated successfully");
-    } catch (FeignException e) {
+    } catch (RuntimeException e) {
```

Comment:
mb `SystemUserAuthorizationException`?

Reply:
in this case it's easier to not throw on L58 :) because we have here RE and in parent scope the same RE

Reply:
We still want to raise exceptions for other failures in the process (user creation, perm assignment, etc.). It is only the login here that was causing issues in karate env